CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+* [Breaking] To improve dfuse instrumented `nodeos` binary processing speed, we had to make incompatible changes to data exchange format going out of `nodeos`. This requires you to upgrade your dfuse instrumented `nodeos` binary to latest version (https://github.com/dfuse-io/eos/releases/tag/v2.0.5-dm-12.0). Follow instructions in at https://github.com/dfuse-io/dfuse-eosio/blob/develop/DEPENDENCIES.md#dfuse-instrumented-eosio-prebuilt-binaries to install the latest version for your platform.
```

Comment:
The `DEPENDENCIES.md` file is going to be updated just before doing the actual release, so people follow the guide right now are not using the latest version.

It might be a good consideration to keep the `master` branch the default instead of `develop` so we can make those change directly without impacting people following "released" instruction which are in `master`. This is how `EOSIO/eos` works.

Might be far fetched though, but you must pay attention to this when updating the "docs" (and related files like `DEPENDENCIES.md`).